packages/core/src/internal/motion/animations/cds-tree-item-expand.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { TargetedAnimation } from '../interfaces.js';
+
+export const AnimationTreeItemExpandName = 'cds-tree-item-expand';
+export const AnimationTreeItemExpandConfig: TargetedAnimation[] = [
+  {
```

Comment:
What did you think about the animations? Hard? Easy?

Reply:
I liked using the decorator. It was helpful to see examples of animations already in place in some of the components.